docs/reference/modules/indices/recovery.asciidoc

```diff
@@ -1,10 +1,23 @@
 [[recovery]]
 === Indices Recovery
 
-The following _expert_ setting can be set to manage the recovery policy.
+_Recovery_ is the process used to build a new copy of a shard on a node by
+copying data from the primary. {es} uses this recovery process to rebuild shard
+copies that were lost if a node has failed, and uses the same process when
+migrating a shard copy between nodes to rebalance the cluster or to honor any
+changes to the <<modules-cluster,shard allocation settings>>.
+
+The following _expert_ setting can be set to manage the resources consumed by
+recoveries:
 
 `indices.recovery.max_bytes_per_sec`::
-    Defaults to `40mb`.
+    Limits the total inbound and outbound recovery traffic on each node. If you
+    set this limit too high then there is a risk that ongoing recoveries will
+    consume an excess of bandwidth (or other resources) which could destabilize
+    the cluster. Since this setting applies to each node separately, the total
+    bandwidth that might be used by recoveries is the value of this setting
+    multiplied by half of the number of nodes in the cluster. Defaults to
```

Comment:
I don't understand this sentence and the formula, and find it rather more confusing than helpful. Maybe omit this?

Reply:
I would like somehow to indicate that although `40MB/s` seems like a small number, it's multiplied by the number of nodes, so 8 nodes all recovering at their maximum speed will saturate a 1Gbit network with recoveries. This seems to be a point of confusion.

Reply:
I'm not sure I understand. The rate limiter works both at the sender and receiver side of file chunks to ensure that even for all recoveries combined we only use `40MB/s` on the connections to/from a single node. Depending on the network topology (e.g. not all nodes connected to the same switch), other bandwidth restrictions might exist. I think it's ok to point out that this limit is a per-node thing and does not account for the overall recovery traffic in the cluster, but I would avoid adding this formula which feels more like a "Milchmädchenrechnung" to me.

Reply:
Ok, weakened in d4ab7a8